os-jdg7-launch/added/openshift-launch.sh

```diff
@@ -25,8 +25,25 @@ CONFIGURE_SCRIPTS=(
   /opt/run-java/proxy-options
 )
 
+function runServer() {
+  local instanceDir=$1
+
+  echo "Running $JBOSS_IMAGE_NAME image, version $JBOSS_IMAGE_VERSION"
+
+  exec $JBOSS_HOME/bin/standalone.sh -c clustered-openshift.xml -bmanagement 127.0.0.1 -Djboss.server.data.dir="$instanceDir" ${JBOSS_HA_ARGS} ${JAVA_PROXY_OPTIONS}
+}
+
 source $JBOSS_HOME/bin/launch/configure.sh
 
```

Comment:
Define the init_data_dir function here

Reply:
It would need an equivalent in the A-MQ script, albeit an empty function.

function init_data_dir() {
  // No initialisation needed
}